include/mongoose.hrl

```diff
@@ -35,4 +35,10 @@
 %% Logging mechanism
 -include("mongoose_logger.hrl").
 
-
+-ifdef(OTP_RELEASE). %% this implies 21 or higher
```

Comment:
Where is it defined? Besides, it's not a good name, as it doesn't imply OTP 21 or higher.

Reply:
From [_Erlang OTP 21.0 is released_](http://www.erlang.org/news/123) (original spelling):

> A new predifined macro `OTP_RELEASE` and preprocessor directives `-if` and `-elif`

If it was added in OTP 21 we can be pretty sure it implies OTP 21 or higher.

Reply:
It also [has a value](http://erlang.org/doc/reference_manual/macros.html) so we could make it more explicit